Link "see issue #....."

Some rustc error messages contain a parenthesized reference to a GitHub issue. For example in [this playground](https://play.rust-lang.org/?gist=9d0cd1432a86efa138a118854952342a&version=nightly):

```rust
#[repr(transparent)]
struct S(u8);

fn main() {}
```

```
error[E0658]: the `#[repr(transparent)]` attribute is experimental (see issue #43036)
 --> src/main.rs:1:1
  |
1 | #[repr(transparent)]
  | ^^^^^^^^^^^^^^^^^^^^
  |
```

It would be helpful to transform these into clickable links ([see issue #43036](#)).

--- a/tests/spec/features/highlighting_error_output_spec.rb
+++ b/tests/spec/features/highlighting_error_output_spec.rb
@@ -11,7 +11,7 @@ RSpec.feature "Highlighting the output", type: :feature, js: true do
   scenario "errors are highlighted" do
     within('.output-stderr') do
       expect(page).to have_css '.error', text: 'too many type parameters provided'
-      expect(page).to have_css '.error', text: 'aborting due to previous error'
+      expect(page).to have_css '.error', text: 'aborting due to 2 previous errors'
       expect(page).to have_css '.error', text: 'Could not compile `playground`'
     end
   end
@@ -19,6 +19,12 @@ RSpec.feature "Highlighting the output", type: :feature, js: true do
   scenario "error locations are links" do
     within('.output-stderr') do
       expect(page).to have_link('src/main.rs')
+    end
+  end
+
+  scenario "github see-issues are links" do
+    within('.output-stderr') do
+      expect(page).to have_link('see issue #27812', href: 'https://github.com/rust-lang/rust/issues/27812')
     end
   end
 
@@ -34,6 +40,8 @@ RSpec.feature "Highlighting the output", type: :feature, js: true do
 
   def code
     <<~EOF
+    extern crate syntax;
+
     fn main() {
         drop::<u8, u8>(1);
     }

--- a/ui/frontend/highlighting.ts
+++ b/ui/frontend/highlighting.ts
@@ -7,6 +7,7 @@ export function configureRustErrors(gotoPosition) {
       pattern: /error(\[E\d+\])?:.*\n/,
       inside: {
         'error-explanation': /\[E\d+\]/,
+        'see-issue': /see issue #\d+/,
       },
     },
     'error-location': /-->.*\n/,
@@ -19,6 +20,12 @@ export function configureRustErrors(gotoPosition) {
       const [errorCode] = errorMatch;
       env.tag = 'a';
       env.attributes.href = `https://doc.rust-lang.org/error-index.html#${errorCode}`;
+    }
+    if (env.type === 'see-issue') {
+      const errorMatch = /\d+/.exec(env.content);
+      const [errorCode] = errorMatch;
+      env.tag = 'a';
+      env.attributes.href = `https://github.com/rust-lang/rust/issues/${errorCode}`;
     }
     if (env.type === 'error-location') {
       const errorMatch = /(\d+):(\d+)/.exec(env.content);

